ghcup-vanilla-0.0.8.yaml

```diff
@@ -4329,14 +4329,113 @@ ghcupDownloads:
+            unknown_versioning: *ghc9101-x86_64-deb11
+          Linux_Fedora:
+            '>= 33':
+              dlHash: eef58297f48cae3d8003fd6e8bc295b6e42fe982ad902eada99a72d559384cab
+              dlSubdir: ghc-9.10.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-fedora33-linux.tar.xz
+            unknown_versioning: *ghc9101-x86_64-centos7
+          Linux_Mint:
+            < 20: &ghc9101-x86_64-ubuntu18_04
+              dlHash: 228241a183e55dd7fcb3988b3f4cd19062df08ad85373b7451df03ba64eb9da2
+              dlSubdir: ghc-9.10.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc9101-x86_64-ubuntu20_04
+              dlHash: ae3be406fdb73bd2b0c22baada77a8ff2f8cde6220dd591dc24541cfe9d895eb
+              dlSubdir: ghc-9.10.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-ubuntu20_04-linux.tar.xz
+          Linux_RedHat:
+            unknown_versioning: *ghc9101-x86_64-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 18 ): *ghc9101-x86_64-deb9
+            ( >= 18 && < 19 ): *ghc9101-x86_64-ubuntu18_04
+            unknown_versioning: *ghc9101-x86_64-ubuntu20_04
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: 853909835ea9de6db39a0db0d52c01e7f200c816e925895199a1e62ad7c5901f
+              dlSubdir: ghc-9.10.1-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-rocky8-linux.tar.xz
+          Windows:
+            unknown_versioning:
+              dlHash: 8bac01906ec2fa5c10c730b5ee5b8165654d654dbaf25ba9d3c42f8e26484c6a
+              dlSubdir: ghc-9.10.1-x86_64-unknown-mingw32
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-x86_64-unknown-mingw32.tar.xz
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlHash: ffaf83b5d7a8b2c04920c6e3909c0be21dde27baf380d095fa27e840a3a2e804
+              dlSubdir: ghc-9.10.1-aarch64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-aarch64-apple-darwin.tar.xz
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: e6df50e62b696e3a8b759670fc79207ccc26e88a79a047561ca1ccb8846157dd
+              dlSubdir: ghc-9.10.1-aarch64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-aarch64-deb10-linux.tar.xz
+      viChangeLog: https://downloads.haskell.org/~ghc/9.10.1/docs/users_guide/9.10.1-notes.html
+      viReleaseDay: '2024-05-10'
+      viSourceDL:
+        dlHash: bf386a302d4ee054791ffd51748900f15d71760fd199157922d120cc1f89e2f7
+        dlSubdir: ghc-9.10.1
+        dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-src.tar.xz
+      viTags:
+      - Latest
```

Comment:
In the vanilla channel `Recommended` and `Latest` usually goes together. Unless there's a reason not to?